doc/nrf/protocols/wifi/mem_requirements.rst

```diff
@@ -0,0 +1,50 @@
+.. _ug_wifi_mem_req:
+
+Memory requirements for Wi-Fi Applications
+==========================================
+
+Code and RAM memory footprint requirements differ depending on the selected platform and the application example.
+
+The following tables list memory requirement values for selected Wi-Fi samples.
+
+Footprint values are provided in kilobytes (KB).
+
+.. tabs::
+
+   .. tab:: nRF52840 DK
+
+      The following table lists memory requirements for samples running on the :ref:`nRF52840 DK <programming_board_names>` (:ref:`nrf52840dk_nrf52840 <zephyr:nrf52840dk_nrf52840>`).
+
+      +------------------------------------+-------------------+---------------------------------+
+      | Sample                             |   Application ROM |   Total RAM (incl. static HEAP) |
+      +====================================+===================+=================================+
+      | :ref:`Scan <wifi_scan_sample>`     |               147 |                              64 |
+      +------------------------------------+-------------------+---------------------------------+
+      | :ref:`STA <wifi_station_sample>`   |               500 |                             219 |
```

Comment:
IMO, mentioning full name of the sample "Station" would be more appropriate here. Same for line 37.

Reply:
STA is a standard acronym for station.

Reply:
Right, but as per my understanding, we generally use STA when we are mentioning the mode. Here, we are referring to the Sample's name and linking to the page.